### ISSUE TITLE
How to use with LSP?

### ISSUE THREAD

**Issue by aloussase** [j-hui/fidget.nvim#233]
I'm using this with lua_ls but it doesn't seem to be doing anything. Here's how I'm setting up fidget:

```lua
  {
    'j-hui/fidget.nvim',
    tag = "v1.0.0",
    config = function()
      require('fidget').setup({})
    end
  }
```

This is using the Lazy plugin manager.

**Comment by j-hui** [comment]
Sorry for the delayed response. For what it's worth, I don't (typically) see any Lua LS notifications when I'm loading small Lua projects---Lua LS [doesn't report progress](https://github.com/LuaLS/lua-language-server/issues/933) when a task completes very quickly. This was previously noticed in https://github.com/j-hui/fidget.nvim/issues/113.

I was able to get a notification when I loaded [telescope.nvim](https://github.com/nvim-telescope/telescope.nvim), but your mileage may vary. I recommend testing Fidget with another language server that is known to reliably send notifications such as rust-analyzer.

By the way, is there a reason you are using v1.0.0?

I recommend using a more recent version, and checking [logs](https://github.com/j-hui/fidget.nvim/blob/1ba38e4cbb24683973e00c2e36f53ae64da38ef5/doc/fidget-option.txt#L841-L842) to see if Fidget is actually receiving the LSP updates (when using something other than Lua LS).
